doc/development/guide/pullrequests.rst

```diff
@@ -45,6 +45,25 @@ merged.
 * **Code factor**:  `Code factor <https://www.codefactor.io/>`_ checks several common code quality
   characteristics. 
 
+To make sure that the formatting and the quality of the new piece of code is up
+to [PEP8 standards](https://www.python.org/dev/peps/pep-0008/) dedicated tools
+(``black`` and ``pylint``) should be run locally.
+
+The PennyLane repository provides top-level files (``.pre-commit-config.yaml``
```

Comment:
Maybe just add the necessary steps:

- pip install pre-commit
- pre-commit install